Source/ACE/Entity/Landblock.cs

```diff
@@ -729,6 +733,13 @@ public void ScheduleItemTransfer(ActionChain chain, ObjectGuid wo, ObjectGuid co
             }
         }
 
+        public bool ItemExistsInWorld(ObjectGuid wo)
```

Comment:
For reference (I don't understand your last post).

When we move items between containers we can identify if it is on the landblock or not by examine ourselves (or items we have exclusive control over).  Starting with a simple scenario:  the only items that can be put in our container are those on the world.

In this case, we're either moving an item between our containers (we can check this safely locally), or it comes from the world, no need to check its on the world to know how to handle it (you can check if its on yourself).

There are weird scenarios where we're grabbing an item from another container (e.g. your home chest, or a corpse), but we've gained exclusive access to that container ahead of time (by opening it), and therefore know to check it as well.

This make sense?